### DISCUSSION TITLE
Restore back the Gitbook Pagelink icon

### DISCUSSION THREAD

**Discussion by ahcsco** [GitbookIO/community#46] [Feature Requests]
### What's your feature/idea?

Please describe here - in as much detail as you can - your idea.
GitBook seems to have changed some of its icons recently in past couple days. 
Noticed the pagelink icon changed from a crisp sharp to this bland doc icon:
<img width="70" alt="Screenshot 2023-01-27 at 15 38 03" src="https://user-images.githubusercontent.com/112613597/215112818-c4e74580-0099-4e70-8ed2-0da98f14a6db.png">
Please restore back the old icon. 

There are seemingly other icons also that have changed for worse. 

### What's your desired outcome?

What would you like to be able to do, if your idea came to life?

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by taszap** [comment]
Thanks for your contribution - we shipped the icons to improve cohesiveness across the product, ensuring greater quality and visual consistency. I would say changing the icons back is a matter of preference than specific feature that could impact and improve multiple users workflows so at the moment it's not something we will consider doing.
